### DISCUSSION TITLE
Embedding subtitles used to remux into mkv

### DISCUSSION THREAD

**Discussion by WarKittens** [JunkFood02/Seal#1212] [Q&A]
Hi. Let me start off by saying I'm not a total computer novice but my skills are about 20 years out of date, and I never dealt with this amount of command line crafting.

When I used to select embed subtitles from YouTube videos, I'd end up with an MKV, which is what I want. Now, it is dumping them directly into the MP4 as a CC track, which doesn't give the end user much control over things like look and feel.

I didn't change anything to my knowledge. Has anyone noticed this? I use the option to select my audio and video before downloading, always have. 

If this is a change that is permanent, would anyone be willing to help me craft a command line to grab premium 1080p (or whatever is the highest available, if higher is available)? My brain just doesn't work that way. I'm in medicine working with COVID nowadays.

Appreciate your time, have a great week.

**Comment by JunkFood02** [comment]
Can you provide a link to a sample video to reproduce this?
